### ISSUE TITLE
How are events captured?

### ISSUE THREAD

**Issue by tommed** [vuedoc/parser#24]
I have the following component:

```javascript
<template>
  <div v-bind:class="formGroupClass">
    <label v-if="label" v-bind:class="labelClass" v-text="label"></label>
    <span v-if="help" class="form-hint" v-text="help"></span>
    <span v-if="error" class="error-message" v-text="error"></span>
    <input v-bind:class="formControlClass"
           :autofocus="autofocus"
           :readonly="readonly"
           :placeholder="placeholder"
           type="text"
           v-bind:value="value"
           v-on:input="$emit('input', $event.target.value)"
           />
  </div>
</template>
```

What do I need to add to the `module.exports` in order to get the "input" event documented?

At the moment, events is always an empty array.

**Comment by demsking** [comment]
Sorry, but the `v-on` directive is not yet supported

**Comment by demsking** [comment]
Implemented on ec0dfa9. It'll be released soon

**Comment by demsking** [comment]
**This project has been moved to https://gitlab.com/vuedoc/parser**
